src/_gettsim_tests/test_interface.py

```diff
@@ -443,8 +375,8 @@ def arbeitsl_geld_2_m_double(arbeitsl_geld_2_m):
         data,
         {},
         functions=[arbeitsl_geld_2_m_double],
-        targets="arbeitsl_geld_2_m_double_hh",
         aggregation_specs=aggregation_specs,
+        targets="arbeitsl_geld_2_m_double_hh",
```

Comment:
```suggestion
        targets="arbeitsl_geld_2_double_m_hh",
```

Unrelated, but let's use the occasion to stick to our own naming conventions.